surface_potential_analysis/kernel/plot.py

```diff
@@ -430,3 +436,30 @@ def plot_isotropic_noise_kernel_1d_x(
     )
     line.set_label(f"{measure} kernel")
     return fig, ax, line
+
+
+def _get_noise_kernel_percentage_error(
+    true_kernel: IsotropicNoiseKernel[
+        TupleBasisLike[*tuple[FundamentalPositionBasis[Any, Any], ...]],
+    ],
+    fitted_kernel: IsotropicNoiseKernel[
+        TupleBasisLike[*tuple[FundamentalPositionBasis[Any, Any], ...]],
+    ],
+) -> ValueList[TupleBasisLike[*tuple[FundamentalPositionBasis[Any, Any], ...]]]:
+    true_data = true_kernel["data"].reshape(true_kernel["basis"].shape)
+    converted = convert_isotropic_kernel_to_basis(fitted_kernel, true_kernel["basis"])
+    return {
+        "basis": true_kernel["basis"],
+        "data": (converted["data"] - true_data) / true_data,
+    }
+
+
+def plot_isotropic_kernel_error(
+    true_kernel: IsotropicNoiseKernel[StackedBasisWithVolumeLike[Any, Any, Any]],
+    fitted_kernel: IsotropicNoiseKernel[StackedBasisWithVolumeLike[Any, Any, Any]],
+) -> tuple[Figure, Axes, Line2D]:
+    """Compare the errors between true kernel and fitted kernel."""
+    percentage_error = _get_noise_kernel_percentage_error(true_kernel, fitted_kernel)[
+        "data"
```

Comment:
Add axis labels

Reply:
now added in plot.py in reduced_caldeira when the function is called